### PR TITLE
feat: add queue to get new notifications

### PR DESCRIPTION
**Pull request by agusaldasoro**
Run a processor worker that reads a SQS queue to store and process local the notifications.

### CODE REVIEW

File: inbox/src/controllers/handlers/internal-handler.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export async function createNotificationsHandler(
   // While this service uses the created_at to order and retrieve notifications
   const createNotificationQuery: SQLStatement = SQL`
```
Comment (hugoArregui) [inline inbox/src/controllers/handlers/internal-handler.ts:25]: what is this handler for?

File: processor/src/components.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    const dbPort = await config.requireString('PG_COMPONENT_PSQL_PORT')
+    const dbHost = await config.requireString('PG_COMPONENT_PSQL_HOST')
+    const dbPassword = await config.requireString('PG_COMPONENT_PSQL_PASSWORD')
+    databaseUrl = `postgres://${dbUser}:${dbPassword}@${dbHost}:${dbPort}/${dbDatabaseName}`
+  }
+
+  const pg = await createPgComponent({ logs, config, metrics })
+
+  // start listener to SQS queue
+  await startListenSQS({ config, logs, pg }) // This worker writes on the database but does not run the migrations
```
Comment (hugoArregui) [inline processor/src/components.ts:35]: I'll move this to service instead, here components are created but not started

File: processor/src/controllers/queue.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    MaxNumberOfMessages: 10,
+    MessageAttributeNames: ['All'],
+    QueueUrl: queueUrl,
+    WaitTimeSeconds: 15,
+    VisibilityTimeout: 3 * 3600 // 3 hours
+  }
+
+  const consumer = new SQSConsumer(sqs, params)
+
+  setInterval(async () => {
```
Comment (hugoArregui) [inline processor/src/controllers/queue.ts:64]: I don't think you need to do this, I think you just `receiveMessages` and that would be executed when there are messages available. We can look into this together, maybe I'm wrong

File: inbox/src/controllers/handlers/internal-handler.ts (hunk 1 above)
Comment (agusaldasoro) [reply inbox/src/controllers/handlers/internal-handler.ts:25]: For testing purposes (:

**Review by hugoArregui — APPROVED** [review]

**Pull request merged**